linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml

```diff
@@ -78,3 +78,9 @@ ocil: |-
     {{% endif %}}
 
 platform: machine
+
+fix: |-
+    Configure {{{ full_name }}} to terminate user sessions after {{{ xccdf_value("var_accounts_tmout") }}} seconds of inactivity.
+
+    Add or edit the following line in "/etc/profile.d/tmout.sh":
+    TMOUT={{{ xccdf_value("var_accounts_tmout") }}}
```

Comment:
I have noticed that in CSV and XSL generated by create_srg_export.py the XCCDF values aren't expanded and there are empty strings instead of them. Is it a known issue? Or what is the recommended way to create fix for parametrized rules?

Reply:
The values must be explicitly set, the default values don't work currently.

Reply:
@Mab879 I have tried to add `var_accounts_tmout=10_min`  into `controls/srg_gpos/SRG-OS-000163-GPOS-00072.yml`, but in the genrated CSV and HTML I get the selector instead of a variable: 

Run the following command to ensure the "TMOUT" value is configured for all userson the system: $ sudo grep TMOUT /etc/profile /etc/profile.d/*.sh The output should return the following: TMOUT=10_min If TMOUT is not set or its value is greater than expected setting then this is a finding.

Configure Red Hat Enterprise Linux 9 to terminate user sessions after 10_min seconds of inactivity.Add or edit the following line in "/etc/profile.d/tmout.sh":TMOUT=10_min

What should I do?

Reply:
See #8612 for details. For now you can set the value, not the value name.

Reply:
Great!